### PR TITLE
Admin panel w/ backend connection

### PR DESCRIPTION
**Pull request by galtouti**

### CODE REVIEW

**Comment by yuvalelarat** [comment]
connected the frontend admin pages to the backend using redux and changed the index.js + store.js files as tomer told me to

there are some comments for the things that i have added in the componenets, they are written in english.

as you asked i used useEffect instead of refetch(),
let me know if there's something wrong/missing.

**Comment by yuvalelarat** [comment]
ready for review

**Review by tomered — CHANGES_REQUESTED** [review]
Please write comments only in English

**Comment by yuvalelarat** [comment]
ready for 2nd review

File: src/components/screens/AdminPage/AdminAlertCenter.jsx [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+                backgroundColor: '#f5f7fa',
+                transition: 'box-shadow 0.3s ease',
+                '&:hover': { boxShadow: '0 8px 20px rgba(0, 0, 0, 0.15)' },
+              }}
+            >
+              <Typography variant="h6" gutterBottom sx={{ fontWeight: 'bold', color: '#3f51b5' }}>
+                System Alerts
+              </Typography>
+
+              {/* רשימת התראות */}
```
Comment (tomered) [inline src/components/screens/AdminPage/AdminAlertCenter.jsx]: Please write comments only in English

**Review by tomered — CHANGES_REQUESTED** [review]

File: src/components/screens/AdminPage/AdminAlertCenter.jsx (hunk 1 above)
Comment (yuvalelarat) [reply src/components/screens/AdminPage/AdminAlertCenter.jsx]: done

**Review by tomered — APPROVED** [review]

**Pull request merged**